ui/src/pages/backoffice/Patron/PatronDetails/components/PatronCurrentBorrowingRequests/PatronCurrentBorrowingRequests.js

```diff
@@ -0,0 +1,146 @@
+import { dateFormatter } from '@api/date';
+import { DocumentTitle } from '@components/Document';
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Loader, Error, ResultsTable } from '@components';
+import { illBorrowingRequest as borrowingRequestApi } from '@api';
+import { invenioConfig } from '@config';
+import { ILLRoutes, BackOfficeRoutes } from '@routes/urls';
+import { SeeAllButton } from '@pages/backoffice/components/buttons';
+import _get from 'lodash/get';
+
+export default class PatronCurrentBorrowingRequests extends Component {
+  componentDidMount() {
+    const patronPid = this.props.patronDetails.user_pid
+      ? this.props.patronDetails.user_pid
+      : null;
+    this.props.fetchPatronCurrentBorrowingRequests(patronPid);
+  }
+
+  seeAllButton = () => {
+    const statuses = invenioConfig.illBorrowingRequests.pendingStatuses.concat(
+      invenioConfig.illBorrowingRequests.requestedStatuses.concat(
+        invenioConfig.illBorrowingRequests.activeStatuses
+      )
+    );
+    const patronPid = this.props.patronDetails.user_pid;
+    const path = ILLRoutes.borrowingRequestListWithQuery(
+      borrowingRequestApi
+        .query()
+        .withPatron(patronPid)
+        .withState(statuses)
+        .qs()
+    );
+    return <SeeAllButton to={path} />;
+  };
+
+  viewDetails = ({ row }) => {
+    return (
+      <Link
+        to={ILLRoutes.borrowingRequestDetailsFor(row.metadata.pid)}
+        data-test={row.metadata.pid}
+      >
+        {row.metadata.pid}
+      </Link>
+    );
+  };
+
+  viewDocument = ({ row }) => {
+    return (
+      <Link
+        to={BackOfficeRoutes.documentDetailsFor(row.metadata.document_pid)}
+        data-test={row.metadata.pid}
+      >
+        <DocumentTitle metadata={row.metadata.document} />
+      </Link>
+    );
+  };
+
+  viewLibrary = ({ row }) => {
+    return (
+      <Link
+        to={ILLRoutes.libraryDetailsFor(row.metadata.library_pid)}
+        data-test={row.metadata.pid}
+      >
+        {row.metadata.library.name}
+      </Link>
+    );
+  };
+
+  viewDate = date => {
+    return <>{_get(date.row, date.col.field) ? dateFormatter(date) : '-'}</>;
```

Comment:
I saw that the `dateFormatter` function does not accept a default value. Why don't you change instead the dateFormatter function to accept it?
```
export function dateFormatter({ col, row }, defaultValue=null) {
    ...
```
and I am not 100% sure, but shouldn't you call it like this `dateFormatter({...date})`?